staff/albanelia0/css-layaut-simple-adaptive/styles/main.css

```diff
@@ -0,0 +1,32 @@
+@import url("mobile.css") only screen and (max-width:768px);
```

Comment:
yo lo he puesto sin url ni only y me funciona, sabes que hace exactamente añadir eso?

Reply:
No exactamente! xD
por ejemplo si utilizas directamente
@media screen and (min-width: 400px) {} o,
@media screen and (max-width: 600px) {}
 tambien te funciona. dentro has de poner el códido que quieras que cambie a partir de esa medida por ejemplo

Reply:
Sí, funciona para indicarle el archivo css que quieres que se ponga en la pantalla mientras tenga esas medidas